shortnames.conf

```diff
@@ -113,3 +113,6 @@
   "python" = "docker.io/library/python"
   # node
   "node" = "docker.io/library/node"
+  # rust
+  "rust" = "ghcr.io/rust-lang/rust"
```

Comment:
https://github.com/rust-lang/docker-rust states that official images are hosted on Docker Hub.